TurkishFilter/sections/general_extensions.txt

```diff
@@ -32,6 +32,8 @@ technopat.net#%#(function() { var w_open = window.open, regex = /goo.gl/; window
 ! END: Clickunders
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/176425
+akakce.com##.billboard_content
```

Comment:
Could you please, move it to `specific.txt` and change this rule a bit so it will has a higher specificity/priority?
For example:
```adblock
akakce.com##body .billboard_content:not(#style_important)
```

The problem is that it doesn't work in apps, because it has lower specificity/priority than style from their CSS.
<details><summary>Screenshot</summary>

![image](https://github.com/AdguardTeam/AdguardFilters/assets/29142494/bdbbd049-73bd-4378-ac24-7586b4c54d7e)

</details>

Reply:
@AdamWr rules in `specific.txt` doesn't includes links to issues, yes?

Reply:
Yes, we usually do not add links to issues in `specific.txt` (except some special cases).